openage/util/dll.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2024-2024 the openage authors. See copying.md for legal info.
+
+"""
+Windows-specific loading of compiled Python modules and DLLs.
+"""
+
+import inspect
+import os
+import sys
+
+# python.dll location
+DEFAULT_PYTHON_DLL_DIR = os.path.dirname(sys.executable)
+
+# openage.dll locations (relative to this file)
+DEFAULT_OPENAGE_DLL_DIRs = [
+    "../../libopenage/Debug",
+    "../../libopenage/Release",
+    "../../libopenage/RelWithDebInfo",
+    "../../libopenage/MinSizeRel",
+]
+
+# nyan.dll locations (relative to this file)
+DEFAULT_NYAN_DLL_DIRS = [
+    "../../../../nyan/build/nyan/Debug",
+    "../../../../nyan/build/nyan/Release",
+    "../../../../nyan/build/nyan/RelWithDebInfo",
+    "../../../../nyan/build/nyan/MinSizeRel",
+    "../../nyan-external/bin/nyan/Debug",
+    "../../nyan-external/bin/nyan/Release",
+    "../../nyan-external/bin/nyan/RelWithDebInfo",
+    "../../nyan-external/bin/nyan/MinSizeRel",
```

Comment:
maybe use nyan here, since that would be the name next to the openage repo when somebody clones the nyan repo

Reply:
`nyan-external` is where nyan is downloaded when `-DDOWNLOAD_NYAN` is set. git nyan is covered by lines 24-27

Reply:
ah right. makes sense :)